### PR TITLE
Fix pylint errors caused by a new stricter version

### PR DESCRIPTION
**Pull request by kralka**

### CODE REVIEW

File: src/sedpack/io/dataset.py [hunk 1]
```diff
@@ -356,6 +356,7 @@ def read_and_decode(self, tf_dataset: TFDatasetT,
     def as_tfdataset(  # pylint: disable=too-many-arguments
```
Comment (jmichelp) [inline src/sedpack/io/dataset.py:356]: Can we remove these pylint directive now or is it still triggering?

Comment (kralka) [reply src/sedpack/io/dataset.py:356]: Different error (still triggering).

Comment (jmichelp) [reply src/sedpack/io/dataset.py:356]: Ok

**Review by jmichelp — APPROVED** [review]

**Comment by jmichelp** [comment]
Regarding yapf: because sedpack has no `.style.yapf` file, it's using the default style.
If we want to be consistent with scaaml repo, either we add the file (so that yapf in command line works the same) or we add `--style=google` to the workflow

**Pull request merged**